src/edges_estimate/foregrounds.py

```diff
@@ -220,6 +220,22 @@ def model(self, **p):
         return p["amp"] * np.sin(2 * np.pi * self.freqs / p["lambda"] + p["phase"])
 
 
+@attr.s
+class Sinusoid_cossin(Foreground):
+    base_parameters = [
+        Parameter("amp0", 0, min=0, max=1, latex=r"A_{\rm sin}"),
+        Parameter("amp1", 0, min=0, max=1, latex=r"A_{\rm cos}"),
+        Parameter("P", 10, min=1, max=30, latex=r"\P_{\rm MHz}"),
```

Comment:
Don't use "\P" in the latex. 

Again, the min/max here are "reasonable" for our intended use, but probably not required. I'd probably make the max infinite.